test/common/database.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const Database = require('../../src/server/database/database-abstraction.js')
+const assert = require('assert')
+
+describe('Database', () => {
+  it('should create a table', (done) => {
+    let db = Database.create()
+
+    db.connect().then(() => {
+      db.table('test').then(() => { done() })
+    })
+  })
+})
+
+describe('Database', function () {
+  it('should put, get, update and remove a document properly', (done) => {
+    let db = Database.create()
```

Comment:
I agree, but this is mainly to leave the database into a consistent state. We'd need to add more API and wrappers to handle this properly.